python/cuml/cluster/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+
+
+set(cython_sources
+    agglomerative.pyx
+    dbscan.pyx
+    hdbscan.pyx
+    kmeans.pyx
+)
+
+set(linked_libraries cuml::${CUML_CPP_TARGET})
+
+if(NOT SINGLEGPU)
+  list(APPEND cython_sources
+       dbscan_mg.pyx
+       kmeans_mg.pyx
+  )
+
+  list(APPEND linked_libraries
+       cumlprims_mg::cumlprims_mg)
+endif()
+
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}"
+  MODULE_PREFIX cluster_
+)
+
+foreach(cython_module IN LISTS RAPIDS_CYTHON_CREATED_TARGETS)
+  set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN;\$ORIGIN/cpp")
```

Comment:
These RPATHs should be pointing to `../library` since [that's where the C++ lib is being installed into](https://github.com/rapidsai/cuml/pull/4818/files#diff-005a07ca82fb46b215f58a3a8f4d6d6cc6ca088ade44e8f15cc2e6a51d1da6f4R95).

Reply:
This also applies to all the other RPATH modifications in the other files.